src/_posts/platform/getting-started/2000-01-01-troubleshooting-ssh.md

```diff
@@ -57,11 +57,14 @@ $ git push scalingo master
 ```
 
 If the branch named `master` does not exist (default branch may be named `main`) you need to create one:
-
 ```bash
-git checkout -b master
+$ git checkout -B master
```

Comment:
The more modern `git` command for this is:
```suggestion
$ git switch --create master
```

Reply:
According to Git documentation `git switch` is still experimental: https://git-scm.com/docs/git-switch#_description

`THIS COMMAND IS EXPERIMENTAL. THE BEHAVIOR MAY CHANGE.`

I'm not in favor to use this one for now.